main.go

```diff
@@ -70,6 +73,7 @@ func main() {
 	region := flag.String("aws-default-region", "", "If set, AWS_DEFAULT_REGION and AWS_REGION will be set to this value in mutated containers")
 	regionalSTS := flag.Bool("sts-regional-endpoint", false, "Whether to inject the AWS_STS_REGIONAL_ENDPOINTS=regional env var in mutated pods. Defaults to `false`.")
 	watchConfigMap := flag.Bool("watch-config-map", false, "Enables watching serviceaccounts that are configured through the pod-identity-webhook configmap instead of using annotations")
+	composeRoleArn := flag.Bool("compose-role-arn", false, "Whether to compose a fully formed role arn when we detect one that is not fully qualified")
```

Comment:
So it has to be the role name and path, right?  Or does just the role name work?  Can you make the comment a little more specific, something like the following (but correct the verbiage if its incorrect):

```suggestion
	composeRoleArn := flag.Bool("compose-role-arn", false, "If true, then the role name and path can be used instead of the fully qualified ARN in the `role-arn` annotation.  In this case, webhook will look up the partition and account ID using instance metadata.  Defaults to `false`.")
```